editor/src/components/inspector/common/property-controls-hooks.ts

```diff
@@ -215,12 +215,14 @@ type FullPropertyControlsAndTargets = {
 const emptyControls: PropertyControls = {}
 
 export function useGetPropertyControlsForSelectedComponents(): Array<FullPropertyControlsAndTargets> {
-  const selectedViews = useRefSelectedViews()
-
+  const selectedViews = useEditorState(
+    (store) => store.editor.selectedViews,
+    'useGetPropertyControlsForSelectedComponents selectedViews',
+  )
   const selectedPropertyControls = useEditorState(
     (store) => {
       let propertyControlsAndTargets: Array<PropertyControlsAndTargets> = []
-      fastForEach(selectedViews.current, (path) => {
+      fastForEach(selectedViews, (path) => {
```

Comment:
Oh wait, the previous `selectedViews` can go in the bin, since this is already inside a `useEditorState` anyway (and `selectedViews` isn't used anywhere else). I have no idea why this was like this before...